app/display/model/src/main/java/org/csstudio/display/builder/model/util/ModelResourceUtil.java

```diff
@@ -309,49 +309,59 @@ private static String doResolveResource(final String parent_display, final Strin
      */
     private static boolean canOpenUrl(final String resource_name)
     {
+        boolean canOpenUrl = false;
         final URL example = getExampleURL(resource_name);
         if (example != null)
         {
             try
             {
                 example.openStream().close();
-                return true;
+                canOpenUrl = true;

```

Comment:
Why change the handling of example:... URLs?

Reply:
I'm using a flag for code coverage